docs/guides/validation_operators_and_actions_intro.rst

```diff
@@ -0,0 +1,98 @@
+.. _validation_operators_and_actions_intro:
+
+
+Validation Operators And Actions Introduction
+==============================================
+
+
+When you want to validate several Expectation Suites together and take an action based on the result of this validation, Validation Operators provide a convenient abstraction for both bundling the validation of multiple expectation suites and the actions that should be taken.
+
+If you are using a Validation Operator that came with GE or was contributed by another developer,
+you can get to a rich set of useful behaviors with very little coding. This is done by editing the operator's configuration in the GE configuration file and by extending the operator in case you want to add new behavior.
+
+Another (future) benefit of Validation Operators is that the grouping of multiple expectation suites will allow compute optimization.
+
+DefaultDataContextAwareValidationOperator
+-----------------------------------------
+
+GE comes with a default operator - DefaultDataContextAwareValidationOperator.
+
+It operates on one batch and evaluates three expectation suites - failures, warnings and quarantine.
+
+This implements business logic that many data practitioners consider useful.
+
+The failure Expectation Suite contains expectations that are considered important enough to justify stopping the pipeline when they are violated.
+
+The expectations in the warnings Expectation Suite are less critical. If a batch does not meet these expectations, it is considered a warning - worth looking into, but the pipeline should be allowed to proceed.
+
+The quarantine Expectation Suite provides a service of filtering out the rows that do not meet the expectations in this suite and returning a clean dataset while saving the non-conforming rows into a quarantine queue for a later review. The pipeline can use the returned dataset (without the non-conforming rows) for next steps.
+
+The quarantine functionality is currently implemented only for Pandas.
+
+
+The operator returns an object that contains sufficient information for the pipeline to decide if it should proceed:
+
+.. code-block:: json
+
+    {
+        "success" : None,
+        "validation_results" : {
+            "failure" : validation result id
```

Comment:
This is not correct. Values under "validation_results" contain (validation_result_id, validation_result) 2-ples. This is a placeholder under we have typed `ValidationResult` objects which can contain their own ids.